taichi/transforms/constant_fold.cpp

```diff
@@ -137,12 +135,10 @@ class ConstantFold : public BasicStmtVisitor {
                       stmt->cast_type,
                       false};
     auto *ker = get_jit_evaluator_kernel(id);
-    auto &current_program = stmt->get_kernel()->program;
-    // save input args to prevent the current kernel from being overridden.
-    ContextArgSaveGuard _(current_program.get_context());
```

Comment:
Delete `class ContextArgSaveGuard` from this file too?

Reply:
Done